Library/Homebrew/livecheck/strategy.rb

```diff
@@ -80,6 +80,50 @@ def from_url(url, livecheck_strategy: nil, regex_provided: nil)
           (strategy.const_defined?(:PRIORITY) ? -strategy::PRIORITY : -DEFAULT_PRIORITY)
         end
       end
+
+      def self.page_headers(url)
+        @headers ||= {}
+
+        return @headers[url] if @headers.key?(url)
+
+        headers = []
+
+        [:default, :browser].each do |user_agent|
+          args = [
+            "--fail",                 # Fail on HTTP errors
```

Comment:
```suggestion
```

We don't need `--fail` here because it's already included by `#curl_args` in this scenario.

Reply:
Right, I just noticed that, removed the commit again.

Reply:
Feel free to push the other changes that were part of that commit, though. I had forgotten that you can do `stdout, _, status = curl_with_workarounds(...)`.